sympy/physics/control/tests/test_lti.py

```diff
@@ -0,0 +1,103 @@
+from sympy import symbols, Matrix, expand, together
+from sympy.physics.control.lti import TransferFunction, is_proper
+
+a, b, g, s, p = symbols('a, b, g, s, p')
+
+
+def test_TransferFunction_create():
+    G = Matrix([1 / s, 1 / (s + 1)])
+    tf1 = TransferFunction(G)
+    assert tf1.G == G
```

Comment:
I think that the most common use case for the class is creating a SISO transfer function, so having to wrap the polynomials in Matrix objects is a bit clunky. Either this class should accept ratios of polynomials in non-matrix form or we also need a `SISOTransferFunction` object.